Simplenote/BiometricAuthentication.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
```

Comment:
Same note as above: Maybe relocate the file to `Classes` ?

Reply:
Thank you, done!